app/src/main/java/com/cmput301f20t21/bookfriends/ui/login/CreateAccountActivity.java

```diff
@@ -0,0 +1,106 @@
+package com.cmput301f20t21.bookfriends.ui.login;
+import android.os.Bundle;
+import android.text.Editable;
+import android.view.View;
+import android.widget.EditText;
+
+import androidx.appcompat.app.AppCompatActivity;
+import com.cmput301f20t21.bookfriends.R;
+import com.google.android.material.textfield.TextInputEditText;
+import com.google.android.material.textfield.TextInputLayout;
+
+public class CreateAccountActivity extends AppCompatActivity {
+    private TextInputLayout usernameLayout;
+    private TextInputLayout passwordLayout;
+    private EditText usernameField;
+    private EditText passwordField;
+    private TextInputLayout confirmLayout;
+    private EditText confirmField;
+    private TextInputLayout emailLayout;
+    private EditText emailField;
+
```

Comment:
It'd be nice if we group these by class. Declare all TextInputLayout then EditText.